srcpkgs/stepmania/template

```diff
@@ -1,28 +1,35 @@
 # Template file for 'stepmania'
 pkgname=stepmania
-version=5.0.12
-revision=2
+version=5.1.0
+revision=1
+archs="i686 x86_64"
 build_style=cmake
 cmake_builddir="Build"
 hostmakedepends="nasm yasm pkg-config git"
 makedepends="libmad-devel libvorbis-devel pcre-devel libjpeg-turbo-devel
-alsa-lib-devel libXrandr-devel libva-devel glew-devel"
+alsa-lib-devel libXrandr-devel libva-devel glew-devel pulseaudio-devel"
 short_desc="Advanced rhythm game"
-maintainer="Michael Aldridge <maldridge@voidlinux.org>"
+maintainer="Jason Manley <jason@jasondavid.tv>"
 license="MIT"
 homepage="http://www.stepmania.com/"
-distfiles="https://github.com/stepmania/stepmania/archive/v${version}.tar.gz"
-checksum=df79bcadd69d4ed60cf560d45386ec275181343495ffd744c3ff8f73c83d4755
+distfiles="https://github.com/stepmania/stepmania/archive/v${version}-b2.tar.gz"
+checksum=2e311df9b661ba287b046ce377eb0b41e426b73e169841df99fc9fe2baa89310
 
-# Upstream has stated that only x86 hardware can meed the performance
-# constraints and that musl is not supported due to interop issues
-# with Windows
-archs="i686 x86_64"
+post_extract() {
+	mv ${XBPS_BUILDDIR}/{stepmania-5.1.0-b2,stepmania-5.1.0}
+
+	sed -i 's/NOT OSS_FOUND AND NOT JACK_FOUND AND NOT ALSA_FOUND AND NOT PULSE_FOUND/NOT OSS_FOUND AND NOT JACK_FOUND AND NOT ALSA_FOUND AND NOT PULSEAUDIO_FOUND/g' ${XBPS_BUILDDIR}/stepmania-5.1.0/StepmaniaCore.cmake
+
+	for i in 16 22 24 32 36 48 64 72 96 128 192 256
+	do
+		vinstall ${XBPS_BUILDDIR}/stepmania-5.1.0/icons/hicolor/${i}x${i}/apps/stepmania-ssc.png 644 usr/share/icons/hicolor/${i}x${i}/apps stepmania-ssc.png
+	done
+	vinstall ${XBPS_BUILDDIR}/stepmania-5.1.0/stepmania.desktop 644 usr/share/applications stepmania.desktop
+}
 
 post_install() {
 	vlicense Docs/Licenses.txt
-
 	mkdir -p ${DESTDIR}/usr/bin
 	cd ${DESTDIR}
-	ln -sf ../../opt/stepmania-5.0/stepmania usr/bin/
+	ln -sf ../../opt/stepmania-5.1/stepmania usr/bin/
```

Comment:
`stepmania-${version%.*}`

Reply:
Actually this one is now wrong.

`${version%.*.b*}` would work, I guess. Not sure what the best one to use is.